src/components/command-bar/commands/search/unified-search/components/dialog-body/chat.module.css

```diff
@@ -0,0 +1,23 @@
+.ask {
+	/* override .linkOrButton tokens */
```

Comment:
💭 Re: `!important` overrides being mentioned in that other thread - if we _do_ want to stick with the override coloration, maybe this "token replacement" approach could be a less "oh no we used `!important`" way to get the overrides in place! 🤔